routes/web.php

```diff
@@ -12,7 +12,38 @@
 | contains the "web" middleware group. Now create something great!
 |
 */
-
 Route::get('/', function () {
     return view('welcome');
 });
+
+Route::get('/register', function () {
+    return view('register');
+});
+
+Route::post('/register', function () {
+    return view('register');
```

Comment:
I don't think this is the way this route will be handled. Delete it and we'll add it later.